choco/themekit.nuspec

```diff
@@ -2,17 +2,17 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>themekit</id>
-    <version>1.2.0</version>
+    <version>1.3.1</version>
     <packageSourceUrl>https://github.com/Shopify/themekit/blob/master/choco</packageSourceUrl>
     <owners>Shopify</owners>
     <title>Shopify Themekit</title>
     <authors>Tim Anema</authors>
-    <projectUrl>https://shopify.github.io/themekit/</projectUrl>
-    <iconUrl>https://shopify.github.io/themekit/choco_assets/shopify-logo.png</iconUrl>
+    <projectUrl>https://shopify.dev/themes/tools/theme-kit/</projectUrl>
+    <iconUrl>https://cdn.shopifycdn.net/static/images/logos/shopify-bag.png</iconUrl>

```

Comment:
The old link is broken, and the latest versions were rejected because of that. This asset seems to follow the [Chocolatey guidelines](https://docs.chocolatey.org/en-us/create/create-packages#package-icon-guidelines).